docker-compose.yml.dist

```diff
@@ -0,0 +1,32 @@
+version: '2'
+services:
+  web:
+    image: fpfis/httpd-php-dev:7.1
+    working_dir: /var/www/html
+    ports:
+      - 8080:8080
+    volumes:
+      - .:/var/www/html # Non Mac users.
+      # - nfsmount:/var/www/html # Mac Users with the nfsmount volume.
+    environment:
+      XDEBUG_CONFIG: "remote_enable=1 remote_host=${DOCKER_HOST_IP} remote_port=9000 idekey=PHPSTORM remote_autostart=1" # Non Mac users.
+      # XDEBUG_CONFIG: "remote_enable=1 remote_host=10.254.254.254 remote_port=9000 idekey=PHPSTORM remote_autostart=1" # Mac users. Run this command in the terminal to map the IP address: sudo ifconfig en0 alias 10.254.254.254 255.255.255.0
```

Comment:
Both mac and non-mac will work with `10.254.254.254` so we can un-comment this line, I'd move the comment above for better readability, but up to you! :)